CONTRIBUTING.md

```diff
@@ -97,15 +109,18 @@ See the table below detailing the release type:
 | Release Candidate | 03    |
 | Release           | 04    |
 
+Example: alpha 5 will be `XX_XX_01_05` and beta 3 will be `XX_XX_02_03`.
+
 #### String naming
+
 Strings `names` in `strings.xml` should follow this format:
 
 | Type                                             | Format            | Example usage              | Actual string        | Other information                                                                                                   |
-|--------------------------------------------------|-------------------|----------------------------|----------------------|---------------------------------------------------------------------------------------------------------------------|
+| ------------------------------------------------ | ----------------- | -------------------------- | -------------------- | ------------------------------------------------------------------------------------------------------------------- |
 | Generic word                                     | $1                | `disagree_or_agree`        | Disagree or agree    | Should only be used if it doesn't fit the below categories                                                          |
 | Action                                           | $1_action         | `apply_action`             | Apply                | Any generic action verb can fit here                                                                                |
 | Preference or popup label<br/>Preference headers | $1_label          | `folders_label`            | Folders              |                                                                                                                     |
 | Preference or popup description                  | $1_description    | `folders_description`      | Row and column count |                                                                                                                     |
 | Preference choice                                | $1_choice         | `off_choice`               | Off                  |                                                                                                                     |
-| Feature string                                   | (feature_name)_$1 | `colorpicker_hsb`          | HSB                  | Feature strings are strings that are confined to a specific feature. Examples include the gesture and color picker. |
+| Feature string                                   | (feature*name)*$1 | `colorpicker_hsb`          | HSB                  | Feature strings are strings that are confined to a specific feature. Examples include the gesture and color picker. |
```

Comment:
CC @JovannMC intended?

Reply:
that was done by prettier - doesn't really matter

Reply:
> It technically _does_ matter, since using asterisks `*` instead of underscores `_` doesn't make sense given the context of the table.

Whoops, I just realised that this is intended and not meant to be italicised. Resolved.